draft-ietf-quic-ack-frequency.md

```diff
@@ -345,8 +345,8 @@ Largest Acked:
 : The Largest Acknowledged value sent in an ACK frame.
 
 Unreported Missing:
-: Packets with packet numbers between the Largest Unacked and Largest Acked that
-  have not yet been received.
+: Packets with packet numbers have not yet been received starting from
+  Largest Acked upto the Largest Unacked.
 
```

Comment:
Maybe "Packets with packet numbers *that* have not yet been received"?

Reply:
But I don't think that captures the corner case. The issue is:

* assume threshold is 2
* endpoint sends ACK[0-3, 5], highest acked number is now 5 
* endpoint receives packet number 6, 7.
* with current text, no ACK is being sent.

Yet in that corner case the endpoint SHOULD send an ACK immediately, because of the hole between 3 and 5. 

My suggested text would be:
~~~
Start of Unreported Range:
: Largest Unacked minus (Reordering Threshold minus 1).
Unreported Missing:
: Packets with packet numbers between the Start of Unreported Range
  and Largest Acked that have not yet been received.
~~~

Reply:
Why is this a corner case?

The text says:

`send an immediate ACK when the gap between the smallest Unreported Missing packet and the Largest Unacked is greater than or equal to the Reordering Threshold value.`

smallest unreported missing = 4
largest unacked = 7
7-4 = 3 > 2

Actually the immediate ACK is already send when packet 6 is received.

Reply:
@huitema can you further explain?

Reply:
This is being fixed in #222